standalone/src/models/product-selection/product-selection-draft/builders.spec.ts

```diff
@@ -0,0 +1,55 @@
+import type {
+  TProductSelectionDraftGraphql,
+  TProductSelectionDraftRest,
+} from '../types';
+import {
+  ProductSelectionDraft,
+  ProductSelectionDraftGraphql,
+  ProductSelectionDraftRest,
+} from './index';
+
+const validateModel = (
+  model: TProductSelectionDraftGraphql | TProductSelectionDraftRest
+): void => {
+  expect(model).toEqual(
+    expect.objectContaining({
+      key: expect.any(String),
+      name: expect.any(Object),
+      mode: expect.any(String),
+      custom: expect.any(Object),
+    })
+  );
+};
+
+describe('ProductSelection model builders', () => {
```

Comment:
also very nit and feel free to ignore if irrelevant but the model is `ProductSelectionDraft`

Reply:
that's a good one, thanks!